### DISCUSSION TITLE
有没有觉得使用体验像小红书？

### DISCUSSION THREAD

**Discussion by Psinary** [share121/inter-knot#5126] [灌水]
![14-51-26-v2-af5925065fff669a85e239caf5a490bf_b](https://github.com/user-attachments/assets/06209362-57d3-4979-a186-b3f1dd66a4fe)
都是卡片图文页面，最尴尬的是，如果不发图，会显示默认图，视觉内容主体上有很大缺失

**Comment by zdsy03** [comment]
红色的桶，能带走灵魂
